### PR TITLE
Fix Taxonomy tests

### PR DESCRIPTION
**Pull request by JarJak**

### CODE REVIEW

File: src/DataFixtures/TaxonomyFixtures.php [hunk 1]
```diff
@@ -41,45 +41,19 @@ private function loadTaxonomies(ObjectManager $manager): void
...
-                );
-
-                $manager->persist($taxonomy);
-                $reference = 'taxonomy_' . $taxonomyDefinition['slug'] . '_' . Str::slug($key);
-                $this->addReference($reference, $taxonomy);
             }
         }
     }
-
-    private function getOptions()
```
Comment (bobdenotter) [inline src/DataFixtures/TaxonomyFixtures.php]: Why remove these?

Comment (JarJak) [reply src/DataFixtures/TaxonomyFixtures.php]: are they needed for something?

Comment (bobdenotter) [reply src/DataFixtures/TaxonomyFixtures.php]: I think it makes sense for the dummy content "fixtures" to have a bunch of tags. Later on, it'll help when we need to add tests for dealing with tags too. So, i'd prefer to keep them.

Comment (bobdenotter) [reply src/DataFixtures/TaxonomyFixtures.php]: @JarJak True, but during development, it's useful to just have a "bunch of tags" available. Can we add a flag to the fixtures code, that allows toggling it?

**Pull request merged**